src/molgenis/capice_resources/train_data_creator/__main__.py

```diff
@@ -0,0 +1,162 @@
+
+
+class TrainDataCreator(Module):
+    def __init__(self):
+        super(TrainDataCreator, self).__init__(
+            program='Train data creator',
+            description='Creator of CAPICE train/test and validation datasets.'
+        )
+
+    @staticmethod
+    def _create_module_specific_arguments(parser):
+        required = parser.add_argument_group('Required arguments')
+        required.add_argument(
+            '-v',
+            '--input-vkgl',
+            type=str,
+            required=True,
+            help='Input location of the public VKGL dataset'
+        )
+        required.add_argument(
+            '-c',
+            '--input-clinvar',
+            type=str,
+            required=True,
+            help='Input location of the ClinVar dataset.'
+        )
+        required.add_argument(
+            '-o',
+            '--output',
+            type=str,
+            required=True,
+            help='Output directory where the files should be placed.'
+        )
+        return parser
+
+    def _validate_module_specific_arguments(self, parser):
+        vkgl = self.input_validator.validate_input_command_line_interface_file(
+            parser.get_argument('input_vkgl'),
+            TSVFileEnums.TSV_EXTENSIONS.value
+        )
+        clinvar = self.input_validator.validate_input_command_line_interface_file(
+            parser.get_argument('input_clinvar'),
+            ('.vcf.gz', '.vcf')
+        )
+        output = self.input_validator.validate_output_command_line_interface_path(
+            parser.get_argument('output')
+        )
+        return {
+            **vkgl,
+            **clinvar,
+            **output
+        }
+
+    def run_module(self, arguments):
+        vkgl = self._read_pandas_tsv(
+            arguments['input_vkgl'],
+            [  # type: ignore
+                Menums.CHROMOSOME.value,
+                Menums.START.value,
+                Menums.SUPPORT.value,
+                Menums.CLASSIFICATION.value
+            ]
+        )
+        parsed_vkgl = VKGLParser().parse(vkgl)
+        clinvar = self._read_vcf_file(
+            arguments['input_clinvar']  # type: ignore
+        )
+        parsed_clinvar = ClinVarParser().parse(clinvar)
+        merge = merge_dataset_rows(parsed_clinvar, parsed_vkgl)
+        del clinvar, parsed_clinvar, vkgl, parsed_vkgl
+        gc.collect()
+
+        ConsensusChecker().check_consensus_clinvar_vkgl_match(merge)
+
+        DuplicateProcessor().process(merge)
+
+        SampleWeighter().apply_sample_weight(merge)
+
+        SVFilter().filter(merge)
+
+        train_test, validation = SplitDatasets().split(merge)
+
+        del merge
+        gc.collect()
+
+        return {
+            DatasetIdentifierEnums.OUTPUT.value: arguments['output'],
+            DatasetIdentifierEnums.TRAIN_TEST.value: train_test,
+            DatasetIdentifierEnums.VALIDATION.value: validation
+        }
+
+    def export(self, output):
+        fake_vcf_header = files(
```

Comment:
Might wanna adjust `setup.py` right away as well in case we might distribute the package in the future: https://setuptools.pypa.io/en/latest/userguide/datafiles.html (just for the src resources ofcourse)

Can locally be tested through `python3 -m build --sdist` and extracting the generated archive in `./dist`.